segmentation-templates-extension/src/SegmentationTemplate.liquid

```diff
@@ -0,0 +1,52 @@
+{%- if flavor contains "react" -%}
+import {
+  reactExtension,
+  useApi,
+  CustomerSegmentationTemplate,
+} from '@shopify/ui-extensions-react/admin';
+
+// The target used here must match the target used in the extension's toml file (./shopify.extension.toml)
+const TARGET = 'admin.customers.segmentation-templates.render';
+
+export default reactExtension(TARGET, () => <App />);
+
+function App() {
+  // The useApi hook provides access to several useful APIs like i18n.
+  const {i18n} = useApi(TARGET);
+
+  const templateQuery = 'number_of_orders = 1 AND products_purchased(id: (product_id)) = true'
+  const templateQueryToInsert = 'number_of_orders = 1 AND products_purchased(id: (',
+
+  return (
+    // The CustomerSegmentationTemplate component provides an API for setting the title, description, icon, query, and query to insert to the segmentation template.
+    <CustomerSegmentationTemplate
+        title={i18n.translate("title")}
+        description={i18n.translate("description")}
```

Comment:
Same with this one, we will soon support localizing the `description` specified in the toml. We should be able to pick up the localized `name` and `description` from the extension config instead of needing these to be set at runtime.